javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -44,13 +44,13 @@ public static void main(String... args) {
         Integer[] ELEMENTS;
 
         /* Only use these for non-mutating operations */
-        final java.util.ArrayList<Integer> javaMutable = new java.util.ArrayList<>();
+        public final java.util.ArrayList<Integer> javaMutable = new java.util.ArrayList<>();
```

Comment:
For displaying the name of the field in the message, without needing to specify it in a String also